### PR TITLE
feat(org): support organization

### PR DESCRIPTION
**Pull request by K0nnyaku**
add organization

### CODE REVIEW

**Review by HsiangNianian — APPROVED** [review]

**Comment by K0nnyaku** [comment]
@fu050409

File: src/utils/organization.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+pub async fn create(db: &Surreal<Client>, org: CreateOrganization) -> Result<Option<Organization>> {
+    Ok(db
+        .create("organization")
+        .content(Organization {
+            id: None,
+            name: org.name,
+            display_name: org.display_name,
+            description: org.description,
+            // should be creator only
+            owner: vec![],
```
Comment (fu050409) [inline src/utils/organization.rs:16]: Consider use:

```suggestion
            owner: vec![org.creator.clone()],
```

File: src/models/organization.rs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    pub updated_at: chrono::NaiveDateTime,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(crate = "rocket::serde")]
+pub struct CreateOrganization {
+    pub name: String,
+    pub display_name: Option<String>,
+    pub description: Option<String>,
+    pub creator: Thing,
```
Comment (fu050409) [inline src/models/organization.rs:27]:
```suggestion
    pub creator: &'data str,
```

`Thing` is hard to deserialize through JSON data, consider use borrowed str and then `Thing::from(("account", org.creator))`to void this.

File: src/routes/index.rs [hunk 3]
```diff
@@ -4,9 +4,8 @@ use crate::{cors::CORS, routes::account};
 use anyhow::Result;
 use rocket::fs::NamedFile;
 use surrealdb::{engine::remote::ws::Ws, opt::auth::Root, Surreal};
-
 use super::problem;
-
+use super::organization;
```
Comment (fu050409) [inline src/routes/index.rs:8]: Have you formmatted the code? I think here should a line break.

**Review by fu050409 — CHANGES_REQUESTED** [review]

**Comment by fu050409** [comment]
@K0nnyaku Mark this pull request as ready after all changes emitted.

**Comment by HsiangNianian** [comment]
WHY "feat(WIP)"

**Comment by fu050409** [comment]
> WHY "feat(WIP)"

Conventional standard, uses `type(WIP)` or tagged with `WIP` tag. If you just want to test a new idea and you are not sure the pull request is intend to be merged, mark your pull request as draft instead of `WIP`.

**Review by fu050409 — APPROVED** [review]

**Comment by HsiangNianian** [comment]
> > WHY "feat(WIP)"
> 
> Conventional standard, uses `type(WIP)` or tagged with `WIP` tag. If you just want to test a new idea and you are not sure the pull request is intend to be merged, mark your pull request as draft instead of `WIP`.
> 

Where is the "Conventional standard"

**Pull request merged**